src/needs-refactoring/components/MappingEvents/MappingEventListPanel.tsx

```diff
@@ -1,18 +1,17 @@
 import { t } from "@transifex/native";
 import Link from "next/link";
 import styled from "styled-components";
-import { useAppContext } from "../../lib/context/AppContext";
-import useCollectionSWR from "../../lib/fetchers/ac/useCollectionSWR";
-import type { MappingEvent } from "../../lib/model/ac/MappingEvent";
-import colors from "../../lib/util/colors";
-import StyledToolbar from "../NodeToolbar/StyledToolbar";
-import { mappingEvent as MappingEventMarkerIcon } from "../icons/markers";
+import { useAppContext } from "~/needs-refactoring/lib/context/AppContext";
+import useCollectionSWR from "~/needs-refactoring/lib/fetchers/ac/useCollectionSWR";
+import type { MappingEvent } from "~/needs-refactoring/lib/model/ac/MappingEvent";
+import colors from "~/needs-refactoring/lib/util/colors";
+import { mappingEvent as MappingEventMarkerIcon } from "~/components/icons/markers";
 import CloseButton from "../shared/CloseButton";
 import StyledMarkdown from "../shared/StyledMarkdown";
 
 export const StyledCloseButton = styled(CloseButton)``;
 
-const StyledMappingEventsToolbar = styled(StyledToolbar)`
+const StyledMappingEventsToolbar = styled.div`
```

Comment:
This is non-functional right now and needs to be migrated later anyway, so I decided to get rid of the Toolbar better now than later :)